src/client/pages/Checkout/CheckoutDetails/CheckoutDetails.tsx

```diff
@@ -32,6 +32,10 @@ export const CheckoutDetails = () => {
 
   const { quoteCartId } = useQuoteCartIdFromUrl()
 
+  useEffect(() => {
```

Comment:
Perhaps a custom hook `useScrollToTop`?

Reply:
This is what React Router docs recommend:

https://v5.reactrouter.com/web/guides/scroll-restoration/scroll-to-top